app/assets/javascripts/patients.coffee

```diff
@@ -23,6 +23,23 @@ valueToNumber = (val) ->
   +val || 0
 
 ready = ->
+  fields = [
+    '#patient_fulfillment_procedure_cost'
+    '#patient_fulfillment_check_number'
+    '#patient_fulfillment_gestation_at_procedure'
+    '#patient_fulfillment_date_of_check'
+    '#patient_fulfillment_procedure_date'
+  ]
+  i = 0
+  field_selectors = ''
+
+  while i<fields.length
```

Comment:
As a 'doesn't write that much js person', I'm not quite sure what this is doing and why, could I trouble you to write a quick descriptive comment about what's going on here?